mpire/utils.py

```diff
@@ -5,10 +5,11 @@
 import time
 from datetime import timedelta
 from multiprocessing import cpu_count
-from multiprocessing.managers import SyncManager
+from multiprocessing.managers import SyncManager as mp_SyncManager
 from multiprocessing.sharedctypes import SynchronizedArray
 from typing import Callable, Collection, Generator, Iterable, List, Optional, Tuple, Union
 
+from multiprocess.managers import SyncManager as mp_dill_SyncManager
```

Comment:
Whoops, very good point. Thnx for catching that!

> NB are any tests run without the extra requirements?

Currently, no. Would you suggest to run the tests a second time, but then without dill related stuff installed? That would require me to add some if/else statements throughout the tests I think. Or mark them with pytest and skip them appropriately

Reply:
https://stackoverflow.com/a/65034094 (example below) suggests it could be possible to temporarily un-import a module, so that might be a way to test whether things that should be able to run without extras actually do.

But I'm not 100% sure if that's as reliable as just running the tests a second time in a venv without those extras. (While skipping tests that require unavailable dependencies.)

Also, if there's any code that automatically chooses a different code-path depending on whether some dependency is available or not, it would be good to run those tests for both cases.

```import sys
from unittest.mock import patch
from importlib import reload

def test_without_panda(self):
    with patch.dict(sys.modules, {'pandas': None}):
        reload(sys.modules['my_module_under_test'])
        # do whatever you want
```

Reply:
Yeah, you're right. I will make this change in a different PR tough.